src/fileio.cpp

```diff
@@ -87,6 +87,8 @@ static void FillValidSearchPaths(bool only_local_path)
 
 	std::set<std::string> seen{};
 	for (Searchpath sp = SP_FIRST_DIR; sp < NUM_SEARCHPATHS; sp++) {
+		if (sp == SP_WORKING_DIR) continue;
+
 		if (only_local_path) {
 			switch (sp) {
 				case SP_WORKING_DIR:      // Can be influence by "-c" option.
```

Comment:
This line is now useless

Reply:
No, it is not. It is the reason this PR works.

Why do you think it is useless?

Reply:
ow, wow, GitHub isn't all that clear it wasn't about the changed text. Right, that explains a bit more.

Yes, that line is now not doing anything. But I am a bit in doubt if we should remove it, as .. it does give information, I am just not sure what information :P